### PR TITLE
Roll Call: Update protocol specification

### PR DESCRIPTION
**Pull request by gnarula**
Please refer to https://docs.google.com/document/d/1fyNWSPzLhM6W9V0VTFf2waMLiJGcscy7wa4bQlLkySM/edit#heading=h.snprjk777bwe

### CODE REVIEW

File: protocol/query/method/message/data/dataCloseRollCall.json [hunk 1]
```diff
@@ -29,12 +29,11 @@
 	"required": ["object", "action", "update_id", "closes", "end", "attendees"],
```
Comment (flmichel) [inline protocol/query/method/message/data/dataCloseRollCall.json:29]: I think "end" should be updated to "closed_at" here. Same thing for the updated fields in dataCreateRollCall.json and dataOpenRollCall.json.

**Review by flmichel — CHANGES_REQUESTED** [review]
Looks good! I just have a minor comment.

**Review by Kar1213 — COMMENTED** [review]
Since now all roll calls that we close have been opened I think we could remove this part of the comment in "closes" field: "or in its absence, the 'id' field of the roll call creation"

**Review by pierluca — APPROVED** [review]
LGTM !

**Review by flmichel — APPROVED** [review]

**Review by Kar1213 — APPROVED** [review]

**Comment by pierluca** [comment]
Following this change, the Fe1 tests are failing.
You might want to update them as needed @Adalsteinnjons @CaiMusso

**Pull request merged**